services_app/src/main/res/layout/conflict_resolver_field_list.xml

```diff
@@ -18,36 +19,33 @@
         android:layout_height="wrap_content"
         android:orientation="vertical" >
 
-        <androidx.appcompat.widget.AppCompatButton
+        <com.google.android.material.button.MaterialButton
             android:id="@+id/take_server"
             android:text="@string/conflict_take_server_updates"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginBottom="24dip"
-            android:layout_weight="50"
+            android:layout_marginBottom="24dp"
             style="@style/ButtonStyle"
             />
 
-        <androidx.appcompat.widget.AppCompatButton
+        <com.google.android.material.button.MaterialButton
             android:id="@+id/take_local"
             android:text="@string/conflict_take_local_updates"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginBottom="24dip"
-            android:layout_weight="50"
+            android:layout_marginBottom="24dp"
             style="@style/ButtonStyle"
             />
 
-        <androidx.appcompat.widget.AppCompatButton
-                android:id="@+id/take_local_with_deltas"
-                android:text="@string/conflict_take_local_with_deltas"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginBottom="24dip"
-                android:layout_weight="50"
```

Comment:
Any justifiable reason for removing layout_weight?

Reply:
> Any justifiable reason for removing layout_weight?

The layout_weight was not affecting the design, and the buttons didn’t require much emphasis, since they aren't major call-to-action buttons like 'Sign in' or 'Verify User.' Hence I have removed it.

Reply:
Great